src/activity_item_handling.cpp

```diff
@@ -1839,9 +1839,9 @@ static std::vector<std::tuple<tripoint, itype_id, int>> requirements_map( player
 
 static void construction_activity( player &p, const zone_data *zone, const tripoint &src_loc,
                                    const activity_reason_info &act_info,
-                                   activity_id activity_to_restore )
+                                   const activity_id &activity_to_restore )
 {
-    const blueprint_options options = dynamic_cast<const blueprint_options &>( zone->get_options() );
+    ( void )zone;
```

Comment:
Instead of this (which is a C-style case and should not be used anyway), comment out the parameter name: `, const zone_data */*zone*/, `

I often see functions starting with `(void) parameter;`, only to find the parameter *is* actually used later. Someone simply forgot to remove the cast to void when they added the usage of that parameter.